v0.5/tools/submission/submission-checker.py

```diff
@@ -0,0 +1,238 @@
+"""
+A checker for mlperf inference submissions
+"""
+
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import argparse
+import collections
+import json
+import logging
+import os
+import re
+import sys
+import time
+
+# pylint: disable=missing-docstring
+
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger("main")
+
+VALID_MODELS = ["ssd-small", "ssd-large", "mobilenet", "resnet", "gnmt"]
+REQUIRED_DIRECTORIES = ["code", "results", "measurements", "systems"]
```

Comment:
Is this checked anywhere?

Reply:
now checking VALID_MODELS. REQUIRED_DIRECTORIES  I don't need ... implicit in the code.